tests/integration/pyramid_tests.py

```diff
@@ -20,7 +21,11 @@
 except ImportError:
     raise unittest.SkipTest("pyramid/webtest is not installed")
 
-from .. import AUTH_TOKEN_PUBLIC_KEY, SERIALIZED_EDGECONTEXT_WITH_VALID_AUTH
+from .. import (
+    AUTH_TOKEN_PUBLIC_KEY,
+    SERIALIZED_EDGECONTEXT_WITH_VALID_AUTH,
+    SERIALIZED_EDGECONTEXT_WITH_NO_AUTH,
+)
```

Comment:
🔕  I _thiiiink_ we might want to do this as multiple imports?  i.e.

```python
from .. import AUTH_TOKEN_PUBLIC_KEY
from .. import SERIALIZED_EDGECONTEXT_WITH_VALID_AUTH
from .. import SERIALIZED_EDGECONTEXT_WITH_NO_AUTH
```

Reply:
just curious, what's the difference? I just do what the linter tells me to  :sob:

Reply:
this is why the linter wants separate lines: https://github.com/asottile/reorder_python_imports#why-this-style

Reply:
In practice, not much, it's just what we do earlier in the file.  I thought `reorder-python-imports` forced this but maybe not for this style of import `from ... import`?